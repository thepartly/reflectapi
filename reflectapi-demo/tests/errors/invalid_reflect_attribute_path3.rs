#[derive(serde::Deserialize, reflectapi::Output)]
struct MyStruct {
    #[reflectapi(output_type = "invalid")]
    field: u32,
}

fn main() {}
