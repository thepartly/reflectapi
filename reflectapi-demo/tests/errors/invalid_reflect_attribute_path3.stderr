error[E0412]: cannot find type `invalid` in this scope
 --> tests/errors/invalid_reflect_attribute_path3.rs:3:32
  |
3 |     #[reflectapi(output_type = "invalid")]
  |                                ^^^^^^^^^ not found in this scope
